### PR TITLE
TEST PRRRRRRrrr

### PR DESCRIPTION
**Pull request by LeZuse**
dd

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -34,16 +34,34 @@ jobs:
...
+  fakedeploy:
+    docker:
+      - image: circleci/node:10
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+      - run: echo uploading assets into cloud .... && sleep 5
+
```
Comment (zuse2) [inline .circleci/config.yml]: nice!

File: .circleci/config.yml [hunk 2]
```diff
@@ -34,16 +34,34 @@ jobs:
...
+          requires:
+            - approve-long
       - build2:
           requires:
             - build
             - approve-long
+      - fakedeploy:
+          requires:
+            - build
+            - build2
```
Comment (zuse2) [inline .circleci/config.yml:67]: nice deploy!

**Review by zuse2 — CHANGES_REQUESTED** [review]
I would change that thing!

**Comment by LeZuse** [comment]
I changed it.

**Review by zuse2 — APPROVED** [review]
LGTM

**Pull request merged**